Core/Core/Models/Context.swift

```diff
@@ -21,6 +21,11 @@ import Foundation
 public enum ContextType: String, Codable {
     case account, course, group, user, section
 
+    public init?(pathComponent: String) {
+        guard pathComponent.last == "s" else { return nil }
```

Comment:
what conditions does this happen?

Reply:
would a comment help here?

Reply:
It goes right along with what's happening below on line 30.